Makefile

```diff
@@ -18,9 +18,8 @@ SHELL := /bin/bash
 VENV_ACTIVATE := .venv/bin/activate
 
 ${VENV_ACTIVATE}: requirements.txt
-	rm -rf .venv
 	python3 -m venv .venv
```

Comment:
Will this recreate .venv or use existing one.

Reply:
It reuses the existing directories/files, but also recreates or updates the timestamp on `.venv/bin/activate`, so it seems to be doing exactly what we need.